src/test/java/com/exasol/adapter/dialects/exasol/AbstractExasolSqlDialectIT.java

```diff
@@ -0,0 +1,596 @@
+                .adapterScript(adapterScript) //
+                .dialectName(EXASOL_DIALECT) //
+                .properties(properties) //
+                .connectionDefinition(this.jdbcConnection) //
+                .build();
+        assertThat(query("SELECT NOW() - INTERVAL '1' MINUTE FROM " + getVirtualTableName(this.virtualSchema, table)),
+                instanceOf(ResultSet.class));
+    }
+
+    @Test
+    void testCreateVirtualSchemaWithoutIgnoreErrorsPropertyThrowsException() throws SQLException {
+        final Table table = createSingleColumnTable("BOOLEAN").insert(true);
+        this.virtualSchema = createVirtualSchema(this.sourceSchema);
+        final SQLException exception = assertThrows(SQLException.class, () -> query(
+                "SELECT NOW() - INTERVAL '1' MINUTE FROM " + getVirtualTableName(this.virtualSchema, table)));
+        assertThat(exception.getMessage(),
+                containsString("Attention! Using literals and constant expressions with datatype "
+                        + "`TIMESTAMP WITH LOCAL TIME ZONE` in Virtual Schemas can produce an incorrect results"));
+    }
+
+    /**
+     * The CastAssertionBuilder is a convenience class that helps formulating cast assertions in a more readable way.
+     */
+    static class CastAssertionBuilder {
+        private final String fromType;
+        private String castToType;
+        private String acceptType;
+        private Object[] inputValues;
+        private final AbstractExasolSqlDialectIT parent;
+
+        public CastAssertionBuilder(final AbstractExasolSqlDialectIT parent, final String from) {
+            this.parent = parent;
+            this.fromType = from;
+        }
+
+        public CastAssertionBuilder to(final String castToType) {
+            this.castToType = castToType;
+            return this;
+        }
+
+        public CastAssertionBuilder accept(final String acceptedType) {
+            this.acceptType = acceptedType;
+            return this;
+        }
+
+        public CastAssertionBuilder input(final Object... input) {
+            this.inputValues = input;
+            return this;
+        }
+
+        public void verify(final Object... expectedValues) {
+            final Table table = prepareInputTable();
+            this.parent.virtualSchema = this.parent.createVirtualSchema(this.parent.sourceSchema);
+            final Builder expectedTable = prepareExpectation(expectedValues);
+            this.parent.assertVsQuery(createCastQuery(table), expectedTable.matches());
+        }
+
+        private Table prepareInputTable() {
+            final Table table = this.parent.createSingleColumnTable(this.fromType);
+            for (final Object sourceValue : this.inputValues) {
+                table.insert(sourceValue);
+            }
+            return table;
+        }
+
+        private Builder prepareExpectation(final Object... expectedValues) {
+            this.acceptType = this.acceptType == null //
```

Comment:
Another nitpick, I personally prefer to use `Optional` in such cases to force the user to acknowledge and deal with cases where the `acceptType` might not be there.
Just a suggestion as well, so feel free to take it or leave it as it is.